rest/model/build.go

```diff
@@ -120,6 +120,7 @@ func (apiBuild *APIBuild) SetTaskCache(tasks []task.Task) {
 			StatusDetails: t.Details,
 			StartTime:     ToTimePtr(t.StartTime),
 			TimeTaken:     t.TimeTaken,
+			TimeTakenMs:   APIDuration(t.TimeTaken.Nanoseconds() / 1000),
```

Comment:
Are we sure that this is the right conversion?  
We also might want to use a function to do this. If we don't yet have one, we should probably create a nanoSecondes to Ms function.

Reply:
I think the conversion from nanoseconds to milliseconds would be nanoSeconds/ 1000000 but I may be misunderstanding something.